SU-SWS/ace-sdssgryphon
# SDSS-445: updating padding on buttons

# READY FOR REVIEW

# Summary
- Changed the Left/right and bottom spacing for buttons.

# Review By (Date)
- Next release

# Urgency
- Normal

# Steps to Test

1. Pull in the change.
2. Verify that the spacing is the following
-- top: 7.5px (ems in code)
-- Padding right-left: MS(1) // 26px
-- Padding bottom: 10px (MS(-2)

# Affected Projects or Products
- Subtheme


# Associated Issues and/or People
[- SDSS-445](https://stanfordits.atlassian.net/browse/SDSS-445)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
